packages/react/src/Column/Column.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+type ColumnTag = 'article' | 'div' | 'section'
+type ColumnGap = 'xs' | 'sm' | 'md' | 'lg' | 'xl'
+
+export type ColumnProps = {
+  /** The element to render the column with. */
+  as?: ColumnTag
+  /** The amount of vertical space between the column’s children. */
+  gap?: ColumnGap
+} & PropsWithChildren<HTMLAttributes<HTMLDivElement>>
```

Comment:
I think this should be `HTMLElement`, because we allow different elements to be rendered

Reply:
This introduces a TypeScript error on `ref`. `Spotlight` and `Grid.Cell` currently suffer from this as well. Let’s address those separately, together with some components that render a `span` and could use `HTMLSpanElement`.